ui-v2/app/templates/dc/services/show.hbs

```diff
@@ -14,27 +14,27 @@
         {{item.Service.Service}}
       </h1>
       <ConsulExternalSource @item={{item.Service}} />
-      <ConsulKind @item={{item.Service}} />
+      <ConsulKind @item={{item.Service}} @withInfo={{true}} />
```

Comment:
The ToolTip Panel should also appear for the `ConsulKind` in app/templates/dc/services/instance.hbs

Reply:
Ah of course! Thanks for the reminder will add that in!